### DISCUSSION TITLE
Backup issue

### DISCUSSION THREAD

**Discussion by Air16** [supabase/supabase#712] [Questions] [answered]
Hi Supabase Team,

Sorry if it is not the right place to talk about a production issue.

Not a big issue, but for your information :
1) the "backup now" button has no effect
2) yesterday there was no daily backup

**Comment by inian** [comment]
Could you give the project id or API endpoint @Air16? Will take a look

**Comment by Air16** [comment]
This is a public site, you can access my account (i'm signin in using github) or give me an email address ?

**Reply by kiwicopple** [reply]
Hey @Air16 - feel free to email beta@supabase.io. Thanks for the bug report!

**Reply by Air16** [reply]
done ! thank you

**Comment by dragarcia** [comment, marked as answer]
Hi @Air16, we've resolved both issues that you have mentioned! The `Backup now` button should now be working as expected and daily backups should be back on its regular schedule. Thanks for highlighting them! 😄

**Reply by Air16** [reply]
thanks !

**Comment by Air16** [comment]
sorry i don't find the button to mark this as answered, feel free to close it.
